### DISCUSSION TITLE
Different behaviour when Delete Folder via API vs APP

### DISCUSSION THREAD

**Discussion by altezza04** [directus/directus#4307] [Archive]
When delete a folder inside the app, it will move all the file inside to parent folder.

while in API, it will move all the fildes to the root folder. 

Should this be the same? or we can have an option in API to decide which folder these files should go to?

**Comment by rijkvanzanten** [comment]
The app does two requests: update all files in the folder to `folder = null`, and then delete the folder.

I don't think there should be too much "custom magic" for just /folders in the API, but it should definitely be a part of https://github.com/directus/directus/issues/2942 (where you can chose what to do on relational deletes, which in turn would mean you'd be able to have the API set the folder to another folder on delete)
